src/main/kotlin/org/rust/lang/core/RsPsiPattern.kt

```diff
@@ -197,6 +197,14 @@ object RsPsiPattern {
                 eq?.elementType == EQ && feature?.textMatches("feature") == true
             }
 
+    val insideCrateTypeAttrValue: PsiElementPattern.Capture<PsiElement> = psiElement(STRING_LITERAL)
+        .withParent(
+            psiElement<RsLitExpr>().withParent(
+                rootMetaItem("crate_type")
+                    .with("innerAttribute") { metaItem -> metaItem.parent is RsInnerAttr }
```

Comment:
Looks like it's wrong condition because it doesn't accept the following code:
```
#![cfg_attr(unix, crate_type = "proc-macro")]
```

I suppose you need use something like
```suggestion
                    .with("innerAttribute") { _, context -> context?.get(META_ITEM_ATTR) is RsInnerAttr }
```

Don't forget to add the corresponding test